tests/test_db.sql

```diff
@@ -1,21 +1,6 @@
-CREATE TABLE thing (
-  id SERIAL,
-  thing_type varchar(50),
-  thing_description TEXT,
-  thing_cost decimal(10,4),
-  thing_in_stock bool
-);
-
-INSERT INTO thing(thing_type, thing_description, thing_cost, thing_in_stock)
-    VALUES('pen', NULL, 50.23, 'f');
-INSERT INTO thing(thing_type, thing_description, thing_cost, thing_in_stock)
-    VALUES('pencil', 'something you write with', 27.50, null);
-INSERT INTO thing(thing_type, thing_description, thing_cost, thing_in_stock)
-    VALUES('marker', NULL, 50.23, 't');
-
-CREATE TABLE test_bool_param (
-  id serial not null,
-  b boolean,
-  primary key(id)
-);
-insert into test_bool_param(b) values(true);
+CREATE TABLE thing (id SERIAL,thing_type varchar(50), thing_description TEXT, thing_cost decimal(10,4), thing_in_stock bool);
+INSERT INTO thing (thing_type, thing_description, thing_cost, thing_in_stock) VALUES ('pen', NULL, 50.23, 'f');
+INSERT INTO thing (thing_type, thing_description, thing_cost, thing_in_stock) VALUES ('pencil', 'something you write with', 27.50, null);
+INSERT INTO thing (thing_type, thing_description, thing_cost, thing_in_stock) VALUES ('marker', NULL, 50.23, 't');
+CREATE TABLE test_bool_param (id serial not null, b boolean,primary key(id));
+INSERT INTO test_bool_param(b) VALUES (true);
```

Comment:
nit: No preference either way, but seems unrelated?

Reply:
Used here: https://github.com/voryx/PgAsync/pull/50/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR51

Reply:
I was referring mostly to the code style changes, as it looks like this does not contain any functional changes, does it? No preference for either style here, only wondered what motivated this change.

Reply:
Ow ah yeah :sweat_smile: . I went through "a few" iterations before opening this PR: https://github.com/WyriHaximus-labs/PgAsync/actions

Some of the work there leaked through. I'll revert it :+1:. (Except the last insert, I'll just capitalize that one :sweat_smile: .)